ecosystem/sep-0006.md

```diff
@@ -1010,6 +1011,22 @@ Name | Type | Description
 * `too_large` -- deposit/withdrawal size exceeded `max_amount`.
 * `error` -- catch-all for any error not enumerated above.
 
+### Refund Object Schema
+
+Name | Type | Description
+-----|------|------------
+`type` | string | One of `partial` or `full`
+`total_refunded` | string | The total amount refunded to the user, in units of `amount_in_asset`. If `type` is `full`, this amount must match `amount_in`.
+`payments` | array | A list of objects containing information on the individual payments made back to the user. The schema for these objects is defined in the section below.
+
+#### Refund Transactions Object Schema
+
+Name | Type | Description
+-----|------|------------
+`refund_payment_id` | string | The payment ID that can be used to identify the refund payment. This is either a Stellar transaction hash or an off-chain payment identifier, such as a reference number provided to the user when the refund was initiated.
```

Comment:
This isn't going to be a unique ID since there's nothing stopping an off-chain system from having overlapping IDs with Stellar transactions. Is that okay?

Reply:
If I understand this scenario, you're saying that an off-chain system could use the Stellar hash as their own ID? Therefore the ID would be both `stellar` and `external`?